NGitLab.Mock/Config/GitLabConfig.cs

```diff
@@ -0,0 +1,50 @@
+﻿namespace NGitLab.Mock.Config
+{
+    /// <summary>
+    /// Describe content of a GitLab server
+    /// </summary>
+    public class GitLabConfig
+    {
+        public GitLabConfig()
+        {
+            Users = new GitLabUsersCollection(this);
+            Groups = new GitLabGroupsCollection(this);
+            Projects = new GitLabProjectsCollection(this);
+        }
+
+        /// <summary>
+        /// User used by default when not specified in extensions methods
+        /// </summary>
+        public string DefaultUser { get; set; }
```

Comment:
Should setters (almost in every object) be internal and only used by the extension methods?